packages/components/ng-ovh-payment-method/src/components/choice/template.html

```diff
@@ -0,0 +1,35 @@
+<oui-skeleton
+    data-size="xl"
+    data-ng-if="$ctrl.loading"></oui-skeleton>
+
+<oui-field
```

Comment:
Just a quick suggestion,maybe you could keep the two radios (disabled while loading) and add skeletons for the label ?

Reply:
Hmm I can disable the two radio buttons, and display skeleton in place of their labels. But, in order to have a skeleton displayed also for the oui-field label, it requires to make two useless disabled oui-radio outside the whole oui-field structure.

For simplicity, I'll make a group of three skeletons, without adding some disabled oui-radio